src/shared/transactionDataHelper.ts

```diff
@@ -145,8 +145,7 @@ export const getTransactionType = ({
   // const involvesCoinTransfer =
   //   txTypes.includes(RawTransactionType.CoinTransfer) &&
   //   !txTypes.includes(RawTransactionType.ContractCreation);
-  const involvesContract =
-    isFromContract || isToContract || createdContract !== undefined;
+  const involvesContract = isFromContract || isToContract || createdContract;
```

Comment:
another way to write is:
```
const involvesContract = isFromContract || isToContract || (createdContract !== undefined &&   createdContract !== null);
```

@fullstackninja864 @pierregee let me know if we prefer this one. thanks

Reply:
fine with  `const involvesContract = isFromContract || isToContract || createdContract;`